### PR TITLE
Conditional rendering angular

### PR DESCRIPTION
**Pull request by mjrdnk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -882,7 +882,57 @@ constructor(props) {
 
 ### Angular
 
-> TODO
+Since Angular 4.0.0, alongside standard ngIf, it is possible to use ngIf;else or ngIf;then;else using `<ng-template>` with an alias `#aliasName`.
+
+ngIf;else example:
```
Comment (feimosi) [inline README.md]: I'd use only `ngIf` in the first example, move the sentence above below the snippet and there show `ngIf;then;else` as it is now. What do you think? Also use `` ` `` for the directive names.

File: README.md [hunk 2]
```diff
@@ -882,7 +882,57 @@ constructor(props) {
...
+
+    <ng-template #registrationSucceeded>
+      <registration-completed></registration-completed>
+    </ng-template>
+
+    <ng-template #registrationFailed>
+      <registration-form></registration-form>
+    </ng-template>
+  `,
+})
```
Comment (feimosi) [inline README.md]: Following the React example, can it be something like:
```html
    <div *ngIf="displaySpecialOffer">
          <special-offer><special-offer />
    </div>

    <div *ngIf="registrationCompleted; then registrationSucceeded; else registrationForm"></div>
 
    <ng-template #registrationSucceeded>
      <registration-completed></registration-completed>
    </ng-template>
 
    <ng-template #registrationForm>
      <registration-form></registration-form>
    </ng-template>
```

File: README.md [hunk 3]
```diff
@@ -882,7 +882,57 @@ constructor(props) {
...
+export class registrationComponent {
+  registrationCompleted: boolean = false;
+
+  constructor() {}
```
Comment (feimosi) [inline README.md]: I guess we can remove the empty `constructor`.

**Review by feimosi — CHANGES_REQUESTED** [review]
Cool, thanks!

**Comment by mjrdnk** [comment]
Morning! How about now? 👍

**Comment by feimosi** [comment]
Looks good :muscle:

**Pull request merged**